cmd/shove/main.go

```diff
@@ -37,7 +38,8 @@ var redisURL = flag.String("queue-redis", "", "Use Redis queue (Redis URL)")
 var webhookWorkers = flag.Int("webhook-workers", 0, "The number of workers pushing Webhook messages")
 
 var webPushVAPIDPublicKey = flag.String("webpush-vapid-public-key", "", "VAPID public key")
-var webPushVAPIDPrivateKey = flag.String("webpush-vapid-private-key", "", "VAPID public key")
+var webPushVAPIDPrivateKey = flag.String("webpush-vapid-private-key", "", "VAPID private key")
+var webPushVAPIDKeysJSON = flag.String("webpush-vapid-keys-json", "", "JSON file containing VAPID keys")
```

Comment:
```suggestion
var webPushVAPIDKeysFile = flag.String("webpush-vapid-keys-file", "", "JSON file containing VAPID keys")
```

(that's consistent with e.g. `-fcm-credentials-file`)